src/types/globals/Window.js

```diff
@@ -35,6 +35,9 @@ declare class Window extends EventTarget {
     element: HTMLElement,
     pseudoEl: ?string
   ) => CSSStyleDeclaration,
+  TextDecoder: typeof TextDecoder,
+  setTimeout: typeof setTimeout,
+  fetch: typeof fetch,
```

Comment:
`TextDecoder` is new, but the rest were needed to add `// @flow` to receive-profile.test.js

Reply:
I wonder if we shouldn't try extend Flow's Window type later. (but this is good enough for now)